test/cppsuite/test_harness/core/configuration.h

```diff
@@ -29,14 +29,16 @@
 #ifndef CONFIGURATION_H
 #define CONFIGURATION_H
 
-#include <algorithm>
 #include <string>
-#include <stack>
+#include <vector>
+
+#include "test_harness/util/debug_utils.h"
 
 extern "C" {
 #include "test_util.h"
 }
 
+namespace test_harness {
 inline std::vector<std::string>
```

Comment:
That can go in the `cxx` too. The `inline` is to avoid a multiple definition linking error since it's in a header, so that can be removed.

Reply:
It's used in runtime_monitor. And if I move implementation to configuration.cxx I get this:
`In file included from ../../../test/cppsuite/test_harness/core/component.h:32:0,
                 from ../../../test/cppsuite/test_harness/runtime_monitor.cxx:30:
../../../test/cppsuite/test_harness/core/configuration.h:43:1: warning: inline function 'std::vector<std::__cxx11::basic_string<char> > test_harness::split_string(const string&, char)' used but never defined
 split_string(const std::string &str, const char delim);`

Reply:
Ah cool. My bad.